src/command_modules/azure-cli-redis/azure/cli/command_modules/redis/commands.py

```diff
@@ -18,26 +18,43 @@ def load_command_table(self, _):
         operations_tmpl='azure.mgmt.redis.operations.patch_schedules_operations#PatchSchedulesOperations.{}',
         client_factory=cf_patch_schedules)
 
+    redis_firewall_rules = CliCommandType(
+        operations_tmpl='azure.mgmt.redis.operations.firewall_rules_operations#FirewallRulesOperations.{}',
+        client_factory=cf_firewall_rule)
+
+    redis_linked_server = CliCommandType(
+        operations_tmpl='azure.mgmt.redis.operations.linked_server_operations#LinkedServerOperations.{}',
+        client_factory=cf_linked_server)
+
     with self.command_group('redis', redis_sdk) as g:
-        g.custom_command('create', 'cli_redis_create', client_factory=cf_redis,
-                         exception_handler=wrong_vmsize_argument_exception_handler)
-        g.command('delete', 'delete')
+        g.custom_command('create', 'cli_redis_create', client_factory=cf_redis)
+        g.command('delete', 'delete', confirmation=True)
         g.custom_command('export', 'cli_redis_export')
         g.command('force-reboot', 'force_reboot')
-        g.custom_command('import-method', 'cli_redis_import_method',
-                         deprecate_info=g.deprecate(redirect='redis import', hide='2.0.34'))
-        g.custom_command('import', 'cli_redis_import_method')
-        g.custom_command('list', 'cli_redis_list')
+        g.command('import-method', 'import_data', deprecate_info=g.deprecate(redirect='redis import'))
```

Comment:
This should handle all of the necessary warnings, which is why your help file doesn't require any of this. I would also suggest adding `hide=True` so that the command doesn't appear when the user runs `-h` but will continue to work.